frontend/src/metabase/collections/components/CollectionMenu/CollectionMenu.tsx

```diff
@@ -48,30 +66,42 @@ export const CollectionMenu = ({
     });
   }
 
-  if (!isRoot && !isPersonal && canWrite && !isInstanceAnalyticsCustom) {
+  if (canMove) {
     items.push({
       title: t`Move`,
```

Comment:
This can be a noun or a verb (I got stung by this recently), so maybe:

```ts
c("The verb, not the noun").t`Move`
```

Reply:
This has been translated for many versions of the app, going to leave this one.